Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -5,6 +5,9 @@
 # common utilities
 ##############################################################################
 def _swapOfflineBSwithOnline(process):
+    import RecoVertex.BeamSpotProducer.onlineBeamSpotESProducer_cfi as _mod
+    process.BeamSpotESProducer = _mod.onlineBeamSpotESProducer.clone()
```

Comment:
on a second thought (vs the earlier proposal for update), why not import as one-line
```suggestion
    from RecoVertex.BeamSpotProducer.onlineBeamSpotESProducer_cfi import onlineBeamSpotESProducer
```

this is a more common way to get here for ES producers, unless there is a clear need to have this specific name.

Reply:
@slava77 your proposal doesn't work. The module needs to be attached to the process for it to be modified.
Since the backport is urgent please consider signing as-is.